### ISSUE TITLE
autoconf doesn't check presence of libusb dev files

### ISSUE THREAD

**Issue by MartinHerren** [steve-m/kalibrate-rtl#14]
autoconf doesn't check presence of libusb dev files, so failing occurs only at compile time.
Like for libfftw and librtlsdr ./configure should check it.

**Comment by viraptor** [comment]
That's an issue of the librtlsdr packaging.
Specifically, libusb is not needed by kalibrate itself. It's needed by librtlsdr and it's included as a dependency.

If your distribution provides a librtlsdr-dev (or similar package) which doesn't depend on libusb-dev, then this bug should be raised with them.

You can see this by running:
```
$ pkg-config --cflags librtlsdr
-I/usr/include/ -I/usr/include/libusb-1.0
```

**Comment by MartinHerren** [comment]
You're correct. Checked the Debian jessie librtlsdr-dev package and it hasn't a dependency on libusb-1.0-0-dev.

Debian jessie-backports, strech and sid have it fixed though, there was a bug-report for that: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=784912

Unfortunately the fix never made it to current stable release.

I guess we can close the bug report here then.
